onchain/rollups/.changeset/grumpy-coins-drop.md

```diff
@@ -0,0 +1,8 @@
+---
+"@cartesi/rollups": major
+---
+
+-   Inputs are now blockchain-agnostic and self-contained blobs.
+-   Removed `sender` from `InputAdded` event.
+-   Library `LibInput` is removed.
+-   Renamed `input` as `payload`
```

Comment:
The first line can be in a dedicated changeset file.
The removal of the `LibInput` can be moved to the `seven-teachers-behave.md` changeset file.
The rest of the changes can be nested in a "Modified the `IInputBox` interface:" changeset file.
Also, the "Renamed `input` as `payload`" change should be instantiated for every entity in the `IInputBox` interface that was affected.